tests/integration/features/sign/request.feature

```diff
@@ -101,7 +101,7 @@ Feature: request-signature
     And I open the latest email to "signer2@domain.test" with subject "LibreSign: There is a file for you to sign"
     And I fetch the signer UUID from opened email
     And as user ""
-    When wait for 1 second
+    When wait for 2 second
```

Comment:
Just to have a note: maybe in future will be better if we use static time - it will be better also for other programmers to understand the code

Reply:
This maybe will be complex because this isn't an unit test.

When is integration test, if we want to use static time, maybe will be necessary do a big work to identify if the environment is executing as integration test and get the time defined by an external way or if is executing normally to follow with dynamic times.